modules/codegen/src/main/scala/com/twilio/guardrail/terms/SwaggerTerm.scala

```diff
@@ -173,6 +174,11 @@ case class RouteMeta(path: String, method: HttpMethod, operation: Operation, sec
 
             p.setRequired(isRequired)
             p.setExtensions(Option(schema.getExtensions).getOrElse(new java.util.HashMap[String, Object]()))
+
+            if (Option(schema.getType()).exists(_ == "file") && contentTypes.contains("application/x-www-form-urlencoded")) {
```

Comment:
Should you use `RouteMeta.ContentType` for this to avoid string compares?

Reply:
Yep, good call